docs/include-cmd-items.rst

```diff
@@ -110,6 +124,9 @@
     d.ratio
     d.resume
```

Comment:
Add a new **TODO** (or move the existing one) if commands are not related to your insertion.